ampersand-form-view.js

```diff
@@ -50,6 +52,14 @@ module.exports = View.extend({
         }
 
         if (opts.values) this._startingValues = opts.values;
+        
+        if (this.validCallback) {
+            this.on('change:valid', function(view, validBool) {
+                this.validCallback(validBool);
```

Comment:
this will call the validCallback when the form goes to invalid (validBool === false), and i believe it should only do so when true